index.css

```diff
@@ -1,3 +1,21 @@
-html {
-  
+html {}
+.board { margin:1em auto; border-collapse:collapse }
+.board td {
+   cursor:pointer;
+   width:100px; height:100px;
```

Comment:
I see where you're going with this, but I'd argue consistency is more important. Someone skimming your code could miss `height` and be really confused debugging some styling.